service/resource/s3bucketv2/delete.go

```diff
@@ -18,7 +18,26 @@ func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange inte
 	if bucketInput.Name != "" {
 		r.logger.LogCtx(ctx, "debug", "deleting S3 bucket")
 
-		_, err := r.clients.S3.DeleteBucket(&s3.DeleteBucketInput{
+		// make sure the bucket is empty.
```

Comment:
Super nitpicky but I think the comments should start with a capital.

Reply:
Sure, fixed :)